### ISSUE TITLE
Ability to have different movie naming profiles within media management

### ISSUE THREAD

**Issue by AndrewTheWookiee** [Radarr/Radarr#7696]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

I add standalone movie folders to my root movies folder as Movies/Movie Name (2000) to sort them alphabetically. However, moves that are a part of a series or shared universe get added to a sub-folder for that series, but within that series sub-folder I name them with the year first because I want those sorted by release year, like Movies/Die Hard Series/(1988) Die Hard.

Currently, I would have to go into Settings/Media Management/Movie Naming and manually change the Movie Folder Format structure if I was adding a different type of movie. 

### Describe the solution you'd like

If there could be an option to save different Movie Naming formats into profiles and have the ability to choose which ones to use when adding movies would be helpful. Something like adding a "Movie Naming" drop-down alongside the Root Folder options on the popup that comes up when you add a movie.

That way when I add a standalone movie I could select a profile I might name as "Single Movie" and it would apply the folder structure I specified for those types of movies, or I could select one I named "Series Movie" and it would select a different folder structure to add the movie with.

### Describe alternatives you've considered

I don't know of any alternative solutions. As of now, I have to remember to go into the settings and change things manually every time. It's not a big deal, but it is a little annoying to have to remember, or go back and rename things when I forgot to change the folder names before adding movies.

### Anything else?

![2022-10-24 12_16_51-Media Management Settings - Radarr — Mozilla Firefox](https://user-images.githubusercontent.com/12468716/197608994-bfc01b45-6adf-4fa3-9393-acc51c24b459.png)

![2022-10-24 12_21_20-Add New Movie - Radarr — Mozilla Firefox](https://user-images.githubusercontent.com/12468716/197609005-3b740347-6b11-4484-a55f-e8ee3f75fcd3.png)

[AB#3965](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3965)

**Comment by bakerboy448** [comment]
Duplicate of #153

**Comment by bakerboy448** [comment]
> get added to a sub-folder for that series, but within that series sub-folder I name them with the year first because I want those sorted by release year, like Movies/Die Hard Series/(1988) Die Hard.

collections as a folder rename variable already exist

**Comment by AndrewTheWookiee** [comment]
I have looked into the collections, but I don't want a 3rd party curated list, I set up my own folders. For instance, I put all the predator and alien movies in the same folder since they share the same in-movie universe. But I only see the alien collection, or the predator collection separately in the Radarr collection list. Further, I do not see the option to change the folder name, only the root folder location. Also, the section in the Radarr wiki for collections is empty.

Apologies if these are dumb questions, but I can't find anything in the collections that solves my problem.
